tests/EnvironmentTest.php

```diff
@@ -65,84 +65,87 @@ public function testGlobals()
         $loader = $this->createMock(LoaderInterface::class);
         $loader->expects($this->any())->method('getSourceContext')->willReturn(new Source('', ''));
 
+        $getGlobals = new \ReflectionMethod(Environment::class, 'getGlobals');
+        $getGlobals->setAccessible(true);
```

Comment:
This is not necessary anymore since PHP 8.1 (see note here: https://www.php.net/manual/en/reflectionmethod.setaccessible.php).